### PR TITLE
ART-1812: Deprecate el8-rebuilds

### PR DESCRIPTION
**Pull request by vfreex**
It is handled by Doozer once https://github.com/openshift/doozer/pull/320 and https://github.com/openshift/ocp-build-data/pull/742 are merged.

### CODE REVIEW

**Review by sosiouxme — APPROVED** [review]
LGTM

**Comment by vfreex** [comment]
/hold
only merge this after the mentioned PRs are merged.

**Pull request merged**
